packages/wrangler/src/queues/cli/commands/create.ts

```diff
@@ -53,9 +54,20 @@ export async function handler(
 	const config = readConfig(args.config, args);
 	const body = createBody(args);
 	try {
-		logger.log(`Creating queue ${args.name}.`);
+		logger.log(`🌀 Creating queue '${args.name}'`);
 		await createQueue(config, body);
-		logger.log(`Created queue ${args.name}.`);
+		logger.log(`✅ Created queue '${args.name}'\n`);
```

Comment:
instead of the multiple `logger.log`s, it might be cleaner to log once and pass in the entire message body to the fn call. So for example:

```suggestion
		logger.log(
		  `✅ Created queue '${args.name}'\n` +
		  "Configure your Worker to send messages to this queue:\n" +
		  "[[queues.producers]]\n" +
		  `queue = "${args.name}"\n` +
		  `binding = "${getValidBindingName(args.name, "queue")}"\n` +
		  "Configure your Worker to consume messages from this queue:\n" +
		  "[[queues.consumers]]\n" +
		  `queue = "${args.name}"`
		);
```

Reply:
unless there's a very particular reason you want to have all these separate fn calls?

Reply:
Oh, no reason! I can stitch them all into one!

Reply:
Done!